ws/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "jsonrpc-ws-server"
+description = "WebSockets server for JSON-RPC"
+version = "6.0.0"
+authors = ["tomusdrw <tomasz@parity.io>"]
+license = "MIT"
+homepage = "https://github.com/ethcore/jsonrpc"
+repository = "https://github.com/ethcore/jsonrpc"
+documentation = "https://ethcore.github.io/jsonrpc/json_ws_server/index.html"
+
+[dependencies]
+log = "0.3"
+jsonrpc-core = { version = "6.0", path = "../core" }
+ws = { git = "https://github.com/tomusdrw/ws-rs" }
```

Comment:
We already have a fork that is extremly out of date with current `ws-rs`. I want to also make a PR upstream with the minimal set of changes I made in my fork.

But first I need to test if everything works in Parity and then will move it there.